content/6_Advanced/Extend_Euclid.mdx

```diff
@@ -34,6 +34,38 @@ ll euclid(ll a, ll b) {
 }
 ```
 
+</CPPSection>
+<PySection>
+
+```py
+def euclid(a: int, b: int) -> int:
+    """Returns integer representing GCD of two positive, non-zero numeric values.
+    """
+
+    # Validate arguments.
+    assert int(a) > 0 and int(b) > 0, "Arguments must be positive, non-zero numeric values."
+
+    # Create tuple object from 'a' and 'b.'
+    ab: list[int] = [int(a), int(b)]
```

Comment:
Easier to do an "in-place" swap in Python using a function with a list.  I can just return values, I suppose.

The code is like this because I wanted to just mimic what was already being done with the C++ version.